test/suite/test_stat08.py

```diff
@@ -41,6 +42,13 @@ class test_stat08(wttest.WiredTigerTestCase):
         READ_TIME : "session: page read from disk to cache time (usecs)"}
```

Comment:
While we're in this file, the constants `4000` and `4003` should be replaced by `wiredtiger.stat.session.bytes_read` and `wiredtiger.stat.session.read_time`.

Reply:
Fixed. It required adding support for `wiredtiger.stat.session` but that is a good improvement.